java/spacewalk-java.changes

```diff
@@ -1,3 +1,4 @@
+- Do not execute immediately Package Refresh action for the SSH minion
```

Comment:
Don't you need a bsc here? @srbarrios mentioned https://github.com/SUSE/spacewalk/issues/20500?

Reply:
Yes, it was created after I wrote the changelog. I will add it.